app/src/components/v-select/v-select.vue

```diff
@@ -370,31 +379,26 @@ function useDisplayValue() {
 							otherVal.value,
 						)
 					"
-					@click.stop
 				>
-					<v-list-item-icon>
+					<v-list-item-content>
 						<v-checkbox
 							:model-value="modelValue || []"
 							:value="otherVal.value"
-							@update:model-value="$emit('update:modelValue', $event.length > 0 ? $event : null)"
-						/>
-					</v-list-item-icon>
-					<v-list-item-content>
-						<input
-							v-focus
-							class="other-input"
-							:value="otherVal.value"
-							:placeholder="t('other')"
-							@input="setOtherValue(otherVal.key, ($event.target as any)?.value)"
-							@blur="otherVal.value.length === 0 && setOtherValue(otherVal.key, null)"
+							custom-value
+							:focus-custom-input="otherVal.focus"
+							@update:model-value="$emit('update:modelValue', $event)"
+							@update:value="setOtherValue(otherVal.key, $event)"
+							@blur:custom-input="
+								(otherVal.value === null || otherVal.value?.length === 0) && setOtherValue(otherVal.key, null)
+							"
```

Comment:
This is getting to a complexity where it's better to have it as a function in the `script` block imo